README.md

```diff
@@ -2,31 +2,58 @@
 
 Implementation of a [TileSource](https://openseadragon.github.io/docs/OpenSeadragon.TileSource.html) for [OpenSeadragon](https://openseadragon.github.io/) based on [geotiff.js](https://geotiffjs.github.io/), enabling local and remote TIFF files to be viewed without using an image server.
 
+Implements an ES6 module extending the original work from [pearcetm/GeoTIFFTileSource](https://github.com/pearcetm/GeoTIFFTileSource). For further information on creating GeoTIFF files and it's requirements, please see the original repository.
 
 
-See it in action at [https://pearcetm.github.io/GeoTIFFTileSource/demo/demo.html](https://pearcetm.github.io/GeoTIFFTileSource/demo/demo.html)
```

Comment:
Having the demo page linked in the readme is still a good idea.

Reply:
Sorry this was my bad - I was planning on creating a react based demo but never got to it. Will add this back

Reply:
A react-based demo in addition to the plain javascript demo would be a great addition!